dev/benchmarks/microbenchmarks/lib/geometry/matrix_utils_transform_rect_bench.dart

```diff
@@ -0,0 +1,139 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
This benchmark covers more than rectangles. Let's remove the `rect` from the file name.

Reply:
Sounds good.  It doesn't test the inverseTransform method, but it benchmarks all the transformFoo methods now, so eliminating rect sounds right.